scripts/chydube.php

```diff
@@ -3,7 +3,7 @@
  //function to desplay intern detials
  function intern($name, $id, $pro_lang, $email)
  {
-     return "Hello World, this is ".$name. " with HNGi7 id ".$id. " using ".$pro_lang." for stage 2 task..".$email;
+     return "Hello World, this is ".$name. " with HNGi7 id ".$id. " using ".$pro_lang." for stage 2 task.".$email;
```

Comment:
All you need is one dot between task and email. you have two right now

Reply:
The other dot is a concatenating dot ... I run a test on my local system it worked